### ISSUE TITLE
view in chrome and safari of verb browser

### ISSUE THREAD

**Issue by monzug** [alpheios-project/webextension#140]
the look of the verb browser is slightly different. I am ok with it but I do understand that we want to have the same style among different browsers
same site in chrome vs safari (http://dcc.dickinson.edu/allen-greenough/Chapters/Chapter-100.html)

![screen shot 2018-10-30 at 3 57 22 pm](https://user-images.githubusercontent.com/41396793/47727543-dbf26d00-dc5c-11e8-9552-0a29b07b0de6.png)

**Comment by balmas** [comment]
I think we don't have explicit styles applied to all of the elements of the inflection browser, because I note that it also looks different on different sites (e.g on http://dcc.dickinson.edu/allen-greenough/Chapters/Chapter-184.html the headers are in italics, while on other sites they are not). So i think the request here is to be explicit about the styles applied to all elements of the inflection browser.  This will still require a fix for #135 for Safari, if possible, but at least we can be consistent about the styles we ask the browser to apply.  Assigning to @irina060981

**Comment by irina060981** [comment]
Fixed
**Safari**
<img width="828" alt="screen shot 2018-11-19 at 18 14 36" src="https://user-images.githubusercontent.com/12377640/48694062-036aa300-ec27-11e8-937f-39caeac0373b.png">

**Comment by irina060981** [comment]
**Firefox**
![image](https://user-images.githubusercontent.com/12377640/48694102-28f7ac80-ec27-11e8-9369-74f8c11f8056.png)

**Comment by balmas** [comment]
ready for testing in build 2.1.0.9

**Comment by monzug** [comment]
one more about the verb browser, Safari build 2.1.0.10 see screenshots
![screen shot 2018-11-21 at 4 15 47 pm](https://user-images.githubusercontent.com/41396793/48850493-1e652080-eda9-11e8-9fa7-143de702d4a2.png)
![screen shot 2018-11-21 at 4 15 58 pm](https://user-images.githubusercontent.com/41396793/48850504-258c2e80-eda9-11e8-9950-c9bbd7810ad0.png)

**Comment by irina060981** [comment]
I have made my tests with the last version of webextension repo (I am not sure that it is the same as the last build) on https://www.loebclassics.com

**Safari - latin selection**
<img width="926" alt="screen shot 2018-11-22 at 10 06 14" src="https://user-images.githubusercontent.com/12377640/48874296-4783c080-ee3e-11e8-8677-500a12f37a08.png">

**Safari - no selection**
<img width="858" alt="screen shot 2018-11-22 at 10 07 33" src="https://user-images.githubusercontent.com/12377640/48874326-769a3200-ee3e-11e8-9050-bf3fd88bbf00.png">

May be it is already fixed with last changes ?

**Comment by balmas** [comment]
@monzug can you retest with build 2.1.0.11? Neither Irina nor I could reproduce this. Thanks!

**Comment by monzug** [comment]
we still have different size in verb browser as reported in the two screenshots above e.g. poesialatina (super small size) and latin library (super size in http://thelatinlibrary.com/lucretius/lucretius2.shtml) I guess we can live with it. @balmas?

**Comment by balmas** [comment]
well, I can live with it, but it probably means we've missed a style that needs to be protected so we should probably look at that. Handing back to @irina060981

**Comment by monzug** [comment]
it's pretty much the same in Chrome so we can pass on this, I think.

**Comment by balmas** [comment]
agree it's not a critical thing, but it does mean we have a style that can be leaked from the site, so as long as we are trying to tackle that I think we can probably fix it.

**Comment by irina060981** [comment]
I have found styles, that should be redefined with the greater weight.
https://github.com/alpheios-project/components/pull/295

**Comment by irina060981** [comment]
![image](https://user-images.githubusercontent.com/12377640/49146934-fea49e00-f34e-11e8-9db5-0c8657717637.png)

![image](https://user-images.githubusercontent.com/12377640/49146948-095f3300-f34f-11e8-98cf-44acaeb903d7.png)

**Comment by balmas** [comment]
please test in build 2.1.0.12

**Comment by monzug** [comment]
YES!!!! font size for verb browser looks good in poesialatina, loeb classic, latin library. Thanks!!!!!
